library/src/android.rs

```diff
@@ -5,12 +5,17 @@ use std::path::{Path, PathBuf};
 
 // https://stackoverflow.com/questions/67087597/is-it-possible-to-use-rusts-log-info-for-tests
 #[cfg(test)]
-use std::println as debug; // Workaround to use println! for logs.
+use std::println as debug;
 
-pub(crate) fn app_data_dir_from_libapp_path(libapp_path: &str) -> anyhow::Result<PathBuf> {
+use crate::UpdateError; // Workaround to use println! for logs.
+
+fn app_data_dir_from_libapp_path(libapp_path: &str) -> Result<PathBuf, UpdateError> {
     // "/data/app/~~7LtReIkm5snW_oXeDoJ5TQ==/com.example.shorebird_test-rpkDZSLBRv2jWcc1gQpwdg==/lib/x86_64/libapp.so"
```

Comment:
For clarity, this is returning the `/data/app/~~7LtReIkm5snW_oXeDoJ5TQ==` directory?